repose-aggregator/extensions/pom.xml

```diff
@@ -18,9 +18,33 @@
         No other filters should be developed here.
     </description>
 
+    <properties>
+        <api-checker.version>1.1.3</api-checker.version>
+    </properties>
+
     <modules>
         <module>api-validator</module>
         <module>simple-rbac</module>
         <module>extensions-filter-bundle</module>
     </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.openrepose</groupId>
+                <artifactId>core-service-api</artifactId>
+                <version>${project.version}</version>
```

Comment:
So push the `org.openrepose` dependencies back into the individual module POM's, but leave the API-Checker one up here?